bears-router.js

```diff
@@ -0,0 +1,84 @@
+const router = require('express').Router();
+
+const knex = require('knex');
+const knexConfig = {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+        filename: './data/lambda.sqlite3',
+    },
+    // debug: true,
+};
+const db = knex(knexConfig);
+
+router.post('/', (req,res) => {
+    if (!req.body.name) {
+        res.status(400).json({error:'missing required field: name'});
+    } else {
+    db('bears').insert(req.body)
+        .then(ids => {
+            const id = ids[0];
+            db('bears').where({id: id}).first()
+                .then(bear => {
+                    res.status(201).json(bear);
+                });
+        })
+        .catch(err => res.status(500).json(err));
+    }
+});
+
+router.get('/', (req,res) => {
+    db('bears')
+        .then(bear => {
+            res.status(200).json(bear);
+        })
+        .catch(err => res.status(500).json(err));
+});
+
+router.get('/:id', (req,res) => {
+    db('bears').where({id: req.params.id}).first()
```

Comment:
same thing as above with shortening your `where` statement up for a small optimization: if you first destructure the id off the req.params before making this call, you could simply say `where({id})` here (would be more useful if you also needed to use that `id` elsewhere in this endpoint, admittedly)